### PR TITLE
Provision fixes

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

**Review by mlohvynenko — APPROVED** [review]

**Review by MykolaSolyanko — APPROVED** [review]

File: src/nodeinfoprovider/nodeinfoprovider.cpp [hunk 1]
```diff
@@ -42,6 +42,7 @@ Error NodeInfoProvider::Init()
         return AOS_ERROR_WRAP(ret);
     }
 
+    mNodeInfo.mName     = "zephyr";
```
Comment (al1img) [inline src/nodeinfoprovider/nodeinfoprovider.cpp:45]: Add AOS_NODE_NAME option to Kconfig and use it

Comment (mykola-kobets-epam) [reply src/nodeinfoprovider/nodeinfoprovider.cpp]: fixed

File: src/nodeinfoprovider/nodeinfoprovider.hpp [hunk 2]
```diff
@@ -60,6 +60,7 @@ class NodeInfoProvider : public iam::nodeinfoprovider::NodeInfoProviderItf {
 
 private:
     static constexpr auto cNodeStatusLen            = 16;
+    static constexpr auto cNodeName                 = CONFIG_AOS_NODE_NAME;
```
Comment (al1img) [inline src/nodeinfoprovider/nodeinfoprovider.hpp:63]: Move to cNodeType

File: src/aoscoreconfig.hpp [hunk 3]
```diff
@@ -72,4 +72,11 @@
  */
 #define AOS_CONFIG_SERVICEMANAGER_SERVICES_DIR CONFIG_AOS_SERVICES_DIR
 
+/**
+ * Configures thread stack alignment.
+ */
+#ifndef AOS_CONFIG_THREAD_STACK_ALIGN
```
Comment (al1img) [inline src/aoscoreconfig.hpp:78]: Why? It is default value now.

File: src/nodeinfoprovider/nodeinfoprovider.cpp [hunk 4]
```diff
@@ -150,6 +150,8 @@ Error NodeInfoProvider::InitNodeID()
     if (auto err = FS::ReadFileToString(cNodeIDFile, mNodeInfo.mNodeID); !err.IsNone()) {
         return AOS_ERROR_WRAP(err);
     }
+
+    mNodeInfo.mNodeID.RightTrim("\r\n");
```
Comment (al1img) [inline src/nodeinfoprovider/nodeinfoprovider.cpp:154]: You can use `Trim` just for case

Comment (mykola-kobets-epam) [reply src/nodeinfoprovider/nodeinfoprovider.cpp]: fixed

File: src/aoscoreconfig.hpp (hunk 3 above)
Comment (mykola-kobets-epam) [reply src/aoscoreconfig.hpp]: commit removed

**Pull request merged**